website/screens/components/grid/code/GridCodePage.tsx

```diff
@@ -0,0 +1,327 @@
+import {
+  DxcFlex,
+  DxcLink,
+  DxcParagraph,
+  DxcTable,
+} from "@dxc-technology/halstack-react";
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Example from "@/common/example/Example";
+import HeaderDescriptionCell from "@/common/HeaderDescriptionCell";
+import basic from "./examples/basic";
+import layout from "./examples/layout";
+
+const sections = [
+  {
+    title: "Props",
+    content: (
+      <DxcTable>
+        <tr>
+          <th>Name</th>
+          <th>Default</th>
+          <HeaderDescriptionCell>Description</HeaderDescriptionCell>
+        </tr>
+        <tr>
+          <td>autoColumns: string</td>
+          <td>
+            <Code>'auto'</Code>
+          </td>
+          <td>
+            Sets the <Code>grid-auto-columns</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-columns"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>autoFlow: 'row' | 'column' | 'row dense' | 'column dense'</td>
+          <td>
+            <Code>'row'</Code>
+          </td>
+          <td>
+            Sets the <Code>grid-auto-flow</Code> CSS property. See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow"
+            >
+              MDN
+            </DxcLink>{" "}
+            for further information.
+          </td>
+        </tr>
+        <tr>
+          <td>autoRows: string</td>
+          <td>
```

Comment:
How do you see to change the order of `autoFlow` and `autoRows` props? I mean, to have `autoColumns`, then `autoRows` and then `autoFlow`.

Reply:
I followed the alphabetical order of props, that's why they appear like that.